package.json

```diff
@@ -26,6 +26,7 @@
 	},
 	"workspaces": ["packages/*", "apps/*"],
 	"volta": {
-		"node": "22.9.0"
+		"node": "22.11.0",
+		"typescript": "5.6.3"
```

Comment:
Why should this use another version than is installed locally in the project?

Reply:
It shouldn't its just what it is automatically pinned to when i pinned typescript